packages/visual-editor/package.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "@contentful/experience-builder-visual-editor",
-  "version": "0.0.2-alpha.26",
+  "name": "@contentful/experiences-visual-editor-react",
+  "version": "0.0.1-alpha.25",
```

```suggestion
  "version": "0.0.1-alpha.1",
```
Similar as on the other packages, I suggest that we don't start a new npmjs package at `alpha.25` but rather at `alpha.1`